### PR TITLE
Wide variety of sample scripts, updated readme to use trial image and docker-compose

### PR DESCRIPTION
**Pull request by cheziyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,88 @@
...
-
-`> docker pull bazzilic/prismadb-mysql`
+**1)** Obtain the Docker Compose file:
 
-And launch it, setting the root password to `toor`:
+Download/save the following YAML Docker Compose file to your machine:
+https://github.com/PrismaDB/PrismaDB/blob/master/scripts/postgres/docker-compose/basic/linux/docker-compose.yml ([Raw File](https://raw.githubusercontent.com/PrismaDB/PrismaDB/master/scripts/postgres/docker-compose/basic/linux/docker-compose.yml))
 
-`> docker run -d --rm -p 3306:3306 --name prismadb-db -e MYSQL_ROOT_PASSWORD=toor bazzilic/prismadb-mysql`
+This Docker Compose file will start a PostgreSQL server with Prisma/DB UDFs, as well as the Prisma/DB Proxy (trial version) for PostgreSQL. Importing/exporting of settings are disabled in the trial build of Prisma/DB Proxy, but other than that, it is a fully featured build.
```
Comment (bazzilic) [inline README.md]: Should it be `is disabled`?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,88 @@
...
+
 INSERT INTO t1 (a, b, c, d, e) VALUES
 ( 1,  2,   3, 'Hello', 'Prisma/DB'),
 (12,  0,   7, 'Test', 'data'),
 ( 0,  2, 123, 'This is encrypted', 'And this is not'),
 (71, 67,  13, 'Last', 'row');
 
 -- Arithmetic operations and search over encrypted values
-SELECT a, b, a + b AS `SUM`, b * a AS `MUL`, (b * a) + b AS `EXPR`
+SELECT a, b, a + b, b * a, (b * a) + b, c, d, e
```
Comment (bazzilic) [inline README.md:69]: Have you checked that the syntax in this sample queries is compatible with Postgres? It was initially written for MySQL

**Review by bazzilic — COMMENTED** [review]
Other than these two comments, looks great!

File: README.md (hunk 2 above)
Comment (cheziyi) [reply README.md:69]: Yup, they are.

File: README.md (hunk 1 above)
Comment (cheziyi) [reply README.md]: Fixed )

**Review by bazzilic — APPROVED** [review]

**Pull request merged**